backend/__tests__/vendorDeleteProperty.test.js

```
@@ -0,0 +1,189 @@
+    User.findByIdAndUpdate = jest.fn().mockResolvedValue({ _id: 'mockUserId' });
+    
+    // Mock file system functions
+    fs.existsSync = jest.fn().mockReturnValue(true);
+    fs.unlink = jest.fn((path, callback) => callback(null));
+    
+    // Set global UPLOADS_DIR
+    global.UPLOADS_DIR = UPLOADS_DIR;
+  });
+  
+  afterEach(() => {
+    delete global.UPLOADS_DIR;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)